### ISSUE TITLE
Report number of active [child?] processes

### ISSUE THREAD

**Issue by richardforth** [richardforth/apache2buddy#15]
For example:

```
[ INFO     ] Your MaxClients setting is 256.
[ INFO     ] Current Apache Process Count is 257 (including the parent pid).
```

or even incorporated in one line:

`[ INFO     ] Your MaxClients setting is 256. Current Count is 257 (including the parent pid).`

be sure to do a grep -v apache2buddy.pl as we found on a previous section of code that grepping for apache2 in the process list will also fish out apache2buddy.pl as well, 

something like

`ps aux | egrep "httpd|apache2" | grep -v apache2buddy | grep -v grep | wc -l`

should do it, but needs some testing.

If people don't like the idea of including the parent I can always minus 1 and report 256 instead (given the above examples). I quite like the idea of leaving the parent process in the count and explaining why we see 257, not 256....... user education.

**Comment by richardforth** [comment]
A change has been committed to staging branch for testing.

**Comment by richardforth** [comment]
Testing went kinda well, (it works) but the display needs prettifying and putting in the right place,

**Comment by richardforth** [comment]
Testing completed:

Output looks like:

```
[ INFO     ] Your MaxClients setting is 256.
[ INFO     ] Current Apache Process Count is 18, including the parent PID.
[ INFO     ] Your ServerLimit setting is 256.
```

(note that number is CYAN or RED if greater than or equal to "maxclients")

Code block:

` curl https://raw.githubusercontent.com/richardforth/apache2buddy/staging/apache2buddy.pl | grep -A10 "Check 14.1"`

----
```
        # Check 14.1
        # Get current number of running apache processes
        # This resolves Issue #15: https://github.com/richardforth/apache2buddy/issues/15
        our $maxclients;
        our $current_proc_count = `ps aux | egrep "httpd|apache2" | grep -v apache2buddy | grep -v grep | wc -l`;
        chomp ($current_proc_count);
        if ($current_proc_count >= $maxclients) {
                if ( ! $NOINFO ) { show_info_box(); print "Current Apache Process Count is ${RED}$current_proc_count${ENDC}, including the parent PID.\n" }
        } else {
                if ( ! $NOINFO ) { show_info_box(); print "Current Apache Process Count is ${CYAN}$current_proc_count${ENDC}, including the parent PID.\n" }
        }
```

**Comment by richardforth** [comment]
Merged this change from staging to master branch
